cli/template/extras/src/server/db/drizzle-schema-auth.ts

```diff
@@ -0,0 +1,108 @@
+import { relations, sql } from "drizzle-orm";
+import {
+  index,
+  int,
+  mysqlTableCreator,
+  primaryKey,
+  serial,
+  text,
+  timestamp,
+  uniqueIndex,
+  varchar,
+} from "drizzle-orm/mysql-core";
+import { type AdapterAccount } from "next-auth/adapters";
+
+/**
+ * This is an example of how to use the multi-project schema feature of Drizzle ORM. Use the same
+ * database instance for multiple projects.
+ *
+ * @see https://orm.drizzle.team/docs/goodies#multi-project-schema
+ */
+export const mysqlTable = mysqlTableCreator((name) => `project1_${name}`);
+
+export const example = mysqlTable(
+  "example",
+  {
+    id: serial("id").primaryKey(),
```

Comment:
We will change the documentation examples for MySQL to use bigint().primaryKey().autoincrement() instead of serial. Using SERIAL may not explicitly convey the use of a bigint autoincrement field and could lead to issues. We will also make the same adjustment for PostgreSQL, especially since PostgreSQL is beginning to recommend IDENTITY columns over SERIAL column types. While we won't deprecate these options, we don't want to feature them in our examples or promote them as the recommended way to set up a schema

So it would be great, if create-t3-app will also recommend this id columns:
```ts
id: bigint("id", { mode: "number" }).primaryKey().autoincrement()
```
We will update drizzle orm docs with this recommendation as well